File: test7.pl
#!/usr/bin/perl

use Time::HiRes qw(gettimeofday);
$i = 0;
@array = ();
while ($i < 100000) {
	push(@array, $i);
	$i++;
}

$t0 = gettimeofday();
foreach $line (@array) {

}
print gettimeofday() - $t0;
print "\n";

File: test7.py
#!/usr/bin/python
from array import *
import random
import time
count = 0
array = []
while (count < 100000): 
	array.append(int(count))
	count = count + 1

a = time.time()
for i in array: 
	pass

print time.time() - a
